wpical/src/main/native/cpp/tagpose.cpp

```diff
@@ -5,6 +5,8 @@
 #include <tagpose.h>
 
 namespace tag {
+const double fieldLength = 16.541;
+const double fieldWidth = 8.211;
```

Comment:
Is this truly fixed?  Doesn't it change yearly?

Reply:
I wasn't aware of that. These values are pulled from the CAD of the field. I was under the impression that the field borders don't change.

Reply:
Consider making these units instead of magic doubles, too.

Reply:
We would need to link wpimath for the units library right? @PeterJohnson would you prefer that the field dimensions be passed by the user side?

Reply:
It's approximately 54 x 27 ft, but changes a little year to year. You can look at https://github.com/wpilibsuite/allwpilib/tree/main/fieldImages/src/main/native/resources/edu/wpi/first/fields for a history

Reply:
That's interesting. In that case, it may be worth having a field for the user to pass the field dimensions as parameters, perhaps with a default value of the 2025 field when dimensions are released on kickoff. Would that work?

Reply:
Ahh, I forgot, the field length and width are included in the .json uploaded to wpical as the initial guess tag poses. We can just use the value from there.